.gitignore

```diff
@@ -55,3 +55,6 @@ scripts/jquery-3.2.1.min.js
 
 # Composer dependencies
 /vendor/
+
+#protect Chris from himself
```

Comment:
I wonder how many places my name will be referenced in the project before version 1.0.0 😂